### PR TITLE
Remove default token

### PR DESCRIPTION
**Pull request by annyhe**

### CODE REVIEW

**Comment by annyhe** [comment]
The coverage dropped due to removing tests that checked for default token's existence.

File: config/passportconfig.js [hunk 1]
```diff
@@ -72,14 +72,7 @@ module.exports = (passportModule) => {
...
       })
-      .then((newUser) => {
-        newUserCreated = newUser;
-        return Token.create({
-          name: newUser.name,
-          createdBy: newUser.id,
-        });
-      })
-      .then(() => {
+      .then((newUserCreated) => {
```
Comment (iamigo) [inline config/passportconfig.js]: does this variable still need to be defined on line 68?

Comment (annyhe) [inline config/passportconfig.js]: nope. I will take it out.

Comment (annyhe) [inline config/passportconfig.js]: @iamigo @pallavi2209 any comments or feedback on the rest of the changes?

**Comment by annyhe** [comment]
Created story as per @pallavi2209  comment

**Comment by pallavi2209** [comment]
Why are tests failing now?

**Comment by annyhe** [comment]
Some test setup relied on a default token to be returned in v1/register response. Will replace these tokens with generated tokens.

**Comment by annyhe** [comment]
@pallavi2209 Updated the tests so they don't rely on default tokens. Please review and comment.

**Comment by annyhe** [comment]
These tests are unrelated to the changes and seem to be flapping. I will run them again.

**Comment by annyhe** [comment]
Tests passed. Please review @pallavi2209 @iamigo

File: tests/api/v1/globalconfig/post.js [hunk 2]
```diff
@@ -41,17 +44,16 @@ describe(`api: POST ${path}`, () => {
...
+        username: uname,
+        email: uname,
         password: 'abcdefghijklmnopqrstuvwxyz',
       })
       .end((err, res) => {
         if (err) {
           done(err);
-        } else {
-          testUserToken = res.body.token;
-          done();
```
Comment (pallavi2209) [inline tests/api/v1/globalconfig/post.js]: Wait, we never changes /v1/register api. Why do we need these changes /v1/register should still return a valid token. This is the token which is created on the fly, without db save.

File: tests/api/v1/globalconfig/get.js [hunk 3]
```diff
@@ -45,16 +48,15 @@ describe(`api: GET ${path}`, () => {
...
+      username: uname,
+      email: uname,
       password: 'abcdefghijklmnopqrstuvwxyz',
     })
     .end((err, res) => {
       if (err) {
         done(err);
       }
 
-      testUserToken = res.body.token;
```
Comment (pallavi2209) [inline tests/api/v1/globalconfig/get.js]: /v1/register api should return this.

File: tests/api/v1/globalconfig/post.js (hunk 2 above)
Comment (annyhe) [inline tests/api/v1/globalconfig/post.js]: Interesting. In the docs the response body to POST to v1/register does not include token. I will double check whether it does on api.

Comment (annyhe) [inline tests/api/v1/globalconfig/post.js]: The tokens are returned on POST v1/register. I added them back in the tests. Please review.

**Comment by annyhe** [comment]
Any comments @iamigo?

**Comment by annyhe** [comment]
This pull request cannot be merged via command line due to the slight decline in coveralls. I forgot the github password so cannot modify the repo settings.
We can merge this via command line after temporarily making master push-able from CLI.

**Comment by iamigo** [comment]
We don't block the ability to merge for code coverage. The problem is failing tests. I restarted the build to see if they pass.

**Pull request merged**